core/src/Network/GRPC/LowLevel/CompletionQueue.hs

```diff
@@ -142,11 +139,7 @@ serverRequestCall rm s scq ccq =
       <*> mgdPayload (methodType rm)
       <*> managed C.withMetadataArrayPtr
     dbug = grpcDebug . ("serverRequestCall(R): " ++)
-    -- On OS X, gRPC gives us a deadline that is just a delta, so we convert
```

Comment:
gRPC now behaves consistently on MacOS and Linux (afaict) so this workaround actually breaks the tests.